### ISSUE TITLE
Status "RE-SUBMITTED" not being used

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/application-manager-web-app#1090]
I notice that when re-submitting an application the status is just set to "SUBMITTED" not "RE-SUBMITTED". 

We should do one of two things:
- Remove "RE-SUBMITTED" as an status enum value
- Add logic so that re-submitted applications do get "RE-SUBMITTED" status. 

The latter would be better in terms of needing less logic in the Activity log/Timeline, but if it's too hard to figure out then we can do it.

@nmadruga do you have a preference?

**Comment by nmadruga** [comment]
> @nmadruga do you have a preference?

I think it needs to be consistent at least. I see why we didn't use it do far, there was no required need to use up until now. So I'm keen to do the second option. But shouldn't we do the same for reviews then? In case yes, would it help with the current impl. of activity_log?

**Comment by CarlosNZ** [comment]
> I think it needs to be consistent at least. I see why we didn't use it do far, there was no required need to use up until now. So I'm keen to do the second option. But shouldn't we do the same for reviews then? In case yes, would it help with the current impl. of activity_log?

Yeah, okay. Let's talk about best way to implement once we've got the activity log in reasonable shape.
